examples/bond/scripts/run/common/common.js

```diff
@@ -21,6 +21,22 @@ const asaDef = {
 
 const tokenMap = new Map();
 
+/**
+ * Execute single transaction or group of transactions (atomic transaction)
+ * @param deployer deployer instance
+ * @param transactions transaction parameters,  atomic transaction parameters
+ * @returns TxnReceipt which includes confirmed txn response along with txID
+ */
+async function tryExecuteTx(deployer, txnParams) {
```

Comment:
Just a bit wonder why we need tryExecuteTx here. because we deployer.executeTx also return error...

Reply:
it returns the error, it doesn't throw it, we need throw to stop the script